src/test/groovy/graphql/kickstart/tools/SchemaParserSpec.groovy

```diff
@@ -266,10 +266,10 @@ class SchemaParserSpec extends Specification {
         when:
         GraphQLSchema schema = SchemaParser.newParser()
                 .schemaString('''\
-                                type Query {
-                                    id: ID!
-                                }
-                            '''.stripIndent())
+                type Query {
```

Comment:
Please match the formatting in the other multi-line String. Should be indented by 4 spaces.

Reply:
This test was failing for me while building locally, that is why I changed the formatting. I've commited a new version where I use stripMargin() instead.